### PR TITLE
Update @nuxtjs/i18n dependency

### PR DESCRIPTION
**Pull request by khushi-kothari**
## Fixes

Related to #3242 by @obulat

## Description

Update @nuxtjs/i18n dependency

## Checklist

- [ ] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [ ] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [ ] My commit messages follow [best practices][best_practices].
- [ ] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.
- [ ] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -152,4 +152,4 @@
     "last 2 versions",
     "not dead"
   ]
-}
+}
```
Comment (obulat) [inline frontend/package.json:155]:
```suggestion
}

```

File: frontend/package.json [hunk 2]
```diff
@@ -92,7 +92,7 @@
     "seeded-rand": "^2.0.1",
     "throttle-debounce": "^5.0.0",
     "uuid": "^8.3.2",
-    "vue": "^2.7.14",
+    "vue": "^2.7.15",
```
Comment (obulat) [inline frontend/package.json:95]:
```suggestion
    "vue": "^2.7.14",
```

File: frontend/package.json [hunk 3]
```diff
@@ -142,8 +142,8 @@
     "vue-i18n-extract": "^2.0.7",
     "vue-jest": "^3.0.7",
     "vue-loader": "^15.10.0",
-    "vue-server-renderer": "^2.7.14",
-    "vue-template-compiler": "^2.7.14",
+    "vue-server-renderer": "^2.7.15",
+    "vue-template-compiler": "^2.7.15",
```
Comment (obulat) [inline frontend/package.json:146]:
```suggestion
    "vue-server-renderer": "^2.7.14",
    "vue-template-compiler": "^2.7.14",
```

**Review by obulat — APPROVED** [review]

**Review by dhruvkb — APPROVED** [review]
The changes look good. Also nice to see a number of `@babel/` packages removed. Thanks for the contribution @khushi-kothari!

**Comment by khushi-kothari** [comment]
> The changes look good. Also nice to see a number of `@babel/` packages removed. Thanks for the contribution @khushi-kothari!

Thank you @dhruvkb for the kind words! I'm glad you liked the changes, and yes, it's always a good feeling to tidy up the dependencies. Thanks for reviewing, and I'm happy to contribute! 😄🙌

**Pull request merged**